modellerScipion/__init__.py

```diff
@@ -27,35 +27,41 @@
 import os, subprocess
 
 from pyworkflow.utils import yellowStr
-import pwem
-
-from pwchem import Plugin as pwchem_plugin
+import pwchem
 
 from .constants import *
 
-
 _version_ = '0.1'
 _logo = "modeller_logo.png"
 _references = ['Webb2016']
 
-MODELLER_DIC = {'name': 'modeller', 'version': '10.4', 'home': 'MODELLER_HOME'}
-
-class Plugin(pwem.Plugin):
+class Plugin(pwchem.Plugin):
     _supportedVersions = [MODELLER_DIC['version']]
 
     @classmethod
     def _defineVariables(cls):
-        """ Return and write a variable in the config file.
-        """
+        """ Return and write a variable in the config file. """
         cls._defineVar("MODELLER_ENV_ACTIVATION", 'conda activate modeller-env')
```

Comment:
shouldn't we change this to the new env name? or if it is not used with the new conf , delete it